doc/api/synopsis.md

```diff
@@ -69,7 +69,7 @@ server.listen(port, hostname, () => {
 });
 ```
 
-Save the file, go back to the terminal window, and enter the following command:
+Save the file and go back to the terminal. Entering the following commands run the file named `hello-world.js`:
```

Comment:
```suggestion
Save the file. Then, in the terminal window, to run the `hello-world.js` file,
enter:
```

Reply:
(See table row for `Enter` at https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui for relevant guidelines/examples on describing user interactions like this.)